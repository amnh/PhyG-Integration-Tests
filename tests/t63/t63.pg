--build(distance, wpgma), inputting fastc file along with tcm
set(seed:1634561640)
read(fastc:"woman.fastc", tcm: "woman-11.tcm")
build(distance, wpgma)
report("t63.csv", data, overwrite)
report("t63.tre", newick, graphs, overwrite)