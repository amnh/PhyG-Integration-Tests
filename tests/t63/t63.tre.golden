(Afrikaans:3.0,(LowSaxon:0.0,((MennoniteLowlandsSaxon:0.0,NorthernLowlandsSaxon:0.0)HTU203:0.0,((SchwyzerdutschSwissGerman:0.0,WesternFrisian:1.0)HTU242:0.0,((Yiddish:0.0,(Hunsrik:0.0,Luxemburgish:0.0)HTU270:0.0)HTU306:0.0,((CentralFranconianRipuarian:0.0,German:0.0)HTU293:1.0,(((Sogdian:2.0,(Latgalian:1.0,Latvian:0.0)HTU302:0.0)HTU333:0.0,(FijiHindi:1.0,((Magahi:1.0,Punjabi:0.0)HTU217:0.0,(Hindi:1.0,Urdu:0.0)HTU231:0.0)HTU247:1.0)HTU371:0.0)HTU377:0.0,(((((KolschColognian:0.0,PennsylvaniaDutch:0.0)HTU340:0.0,((EasternFrieslandLowlandsSaxon:0.0,NorthFrisianOomrang:0.0)HTU239:0.0,((EasternFrisianSaterland:0.0,StellingWervenLowSaxon:0.0)HTU295:0.0,((FrenchFlemish:1.0,(Brabantish:2.0,Limburgish:0.0)HTU262:0.0)HTU279:0.0,(Zeelandic:0.0,(WesternFlemish:0.0,(Dutch:1.0,Flemish:1.0)HTU269:0.0)HTU276:1.0)HTU288:1.0)HTU311:0.0)HTU335:0.0)HTU351:0.0)HTU361:2.0,(((Scots:2.0,((EnglishAmerican:0.0,EnglishBritish:0.0)HTU196:1.0,(EstuaryEnglish:1.0,MiddleEnglish:0.0)HTU238:0.0)HTU258:0.0)HTU305:0.0,(((Sicilian:1.0,((Neapolitan:1.0,(Latin:0.0,Sardinian:0.0)HTU225:0.0)HTU235:0.0,(VenetianBellunese:1.0,(Logudorese:0.0,(Campidanese:1.0,OldItalian:0.0)HTU211:0.0)HTU214:0.0)HTU248:0.0)HTU255:0.0)HTU265:0.0,(Provencal:1.0,(SavoyardFrancoProvençal:0.0,(Friulian:0.0,RumantschGrischun:0.0)HTU229:0.0)HTU252:0.0)HTU281:0.0)HTU291:0.0,(Romanian:0.0,((Walloon:0.0,(French:0.0,MauritianCreole:0.0)HTU202:0.0)HTU226:0.0,((AntilleanCreole:0.0,HaitianCreole:0.0)HTU197:0.0,(LouisianaCreole:0.0,Picard:0.0)HTU240:0.0)HTU246:0.0)HTU259:0.0)HTU319:0.0)HTU324:0.0)HTU353:0.0,(Saraiki:0.0,(Russian:0.0,((Danish:1.0,Icelandic:1.0)HTU261:0.0,(OldFrench:2.0,(Catalan:0.0,RomanschVallader:1.0)HTU241:0.0)HTU277:0.0)HTU322:1.0)HTU343:0.0)HTU362:2.0)HTU366:0.0)HTU382:0.0,((((Konkani:0.0,(TocharianB:0.0,((BalochiSouthern:1.0,(ArmenianWestern:1.0,(ArmenianEastern:0.0,ClassicalArmenian:0.0)HTU237:0.0)HTU243:0.0)HTU298:1.0,(Kashmiri:0.0,OldPrussian:1.0)HTU315:1.0)HTU331:0.0)HTU347:2.0)HTU360:0.0,((AncientIonicGreek:1.0,BelizeanKriol:0.0)HTU292:1.0,(((ResiaSlovenian:1.0,Ukrainian:0.0)HTU284:1.0,((Gothic:0.0,HungarianUralic:0.0)HTU283:1.0,(FinnishUralic:0.0,OldEnglish:2.0)HTU303:0.0)HTU332:0.0)HTU346:1.0,(((Mazanderani:0.0,(Zazaki:0.0,(KurdishCentral:0.0,KurdishKurmanji:0.0)HTU198:0.0)HTU273:0.0)HTU325:0.0,(Welsh:1.0,(Irish:1.0,(OldIrish:0.0,(Cornish:0.0,Manx:0.0)HTU253:1.0)HTU278:1.0)HTU309:0.0)HTU326:0.0)HTU348:0.0,(((OldHighGerman:1.0,((Faroese:0.0,OldNorse:0.0)HTU216:1.0,(Belorussian:0.0,((FerrareseEmiliano:0.0,PiemonteseTurinese:0.0)HTU205:1.0,(LombardPlesio:0.0,RavennateRomagnol:0.0)HTU207:0.0)HTU221:0.0)HTU257:0.0)HTU280:0.0)HTU301:0.0,((Norweigan:2.0,Swedish:0.0)HTU250:1.0,(ProvençalOccitan:1.0,(Italian:0.0,LigurianStella:0.0)HTU199:1.0)HTU274:0.0)HTU308:0.0)HTU336:0.0,((BurgenlandKajkavian:1.0,((((Rusyn:0.0,SerboCroatian:0.0)HTU208:0.0,(Czech:1.0,Slovak:0.0)HTU219:0.0)HTU232:0.0,((Bulgarian:0.0,Macedonian:0.0)HTU200:0.0,(OrlecČakavian:0.0,(OldChruchSlavonicRomanAlphabet:0.0,OldChurchSlavonic:0.0)HTU204:0.0)HTU220:0.0)HTU233:1.0)HTU256:0.0,(LowerSorbian:2.0,(Slovene:1.0,(IstroRomanian:0.0,OrbanićiČakavian:0.0)HTU224:0.0)HTU263:0.0)HTU266:0.0)HTU289:0.0)HTU321:0.0,((PharasaGreek:0.0,SouthernTsakonian:0.0)HTU230:0.0,(AncientGreek:1.0,GreekModern:0.0)HTU260:0.0)HTU328:1.0)HTU337:0.0)HTU350:1.0)HTU364:0.0)HTU369:1.0)HTU373:0.0)HTU378:0.0,(((ClassicalSyriacMiddleAramaicAfroAsiatic:2.0,Luwian:0.0)HTU339:0.0,(((Parachi:0.0,Yaghnobi:0.0)HTU212:0.0,(Persian:0.0,(Elamite:0.0,(MiddlePersian:0.0,Tajik:0.0)HTU209:0.0)HTU227:0.0)HTU251:0.0)HTU300:0.0,(AtongSinoTibetan:0.0,Kashubian:1.0)HTU313:0.0)HTU359:2.0)HTU368:0.0,((Dhatki:0.0,DigorOssetic:2.0)HTU268:1.0,(((Romani:0.0,(GeorgianKartvelian:1.0,TocharianA:0.0)HTU254:0.0)HTU286:1.0,(Oriya:1.0,(Sindhi:1.0,TokPisin:0.0)HTU271:0.0)HTU341:0.0)HTU352:0.0,(((MarwariNorthern:0.0,MarwariSouthern:0.0)HTU201:3.0,(Parkari:2.0,(Avestan:0.0,BrahuiDravidian:2.0)HTU267:0.0)HTU287:0.0)HTU317:0.0,(SanskritVedic:2.0,(Gujarati:1.0,(GujaratiRomanAlphabet:1.0,Marathi:0.0)HTU223:0.0)HTU285:0.0)HTU320:1.0)HTU356:0.0)HTU365:0.0)HTU374:0.0)HTU379:0.0)HTU380:0.0,(((Lithuanian:1.0,((GuineaBissauCreole:1.0,((MeglenoRomanian:0.0,(Aromanian:0.0,Dalmation:0.0)HTU222:0.0)HTU228:0.0,((OldPortuguese:1.0,(PortugueseBrazil:0.0,(Galician:0.0,Portuguese:0.0)HTU206:0.0)HTU210:0.0)HTU213:0.0,(Spanish:0.0,(Aragonese:1.0,Asturian:0.0)HTU236:0.0)HTU244:0.0)HTU264:1.0)HTU299:0.0)HTU323:0.0,(ArchaicLatin:0.0,Vlach:2.0)HTU327:0.0)HTU345:0.0)HTU355:0.0,(Pali:2.0,(Bhojpuri:1.0,(Papiamento:0.0,(Basque:1.0,Breton:1.0)HTU282:0.0)HTU290:1.0)HTU307:0.0)HTU367:0.0)HTU375:0.0,(((Ossetian:0.0,Waziri:1.0)HTU318:1.0,(((MistřiceMoravian:0.0,UpperSorbian:1.0)HTU272:0.0,(Bengali:1.0,(Polish:1.0,Silesian:0.0)HTU249:2.0)HTU310:0.0)HTU312:0.0,(LadinGardenese:0.0,Pashto:1.0)HTU316:0.0)HTU358:1.0)HTU363:0.0,((Samogitian:1.0,Sinhalese:3.0)HTU349:0.0,((((Assamese:0.0,Nepali:0.0)HTU294:0.0,(Hittite:1.0,(SrananTongo:0.0,(JamaicanPatois:0.0,Krio:0.0)HTU215:0.0)HTU234:0.0)HTU304:0.0)HTU330:2.0,((Sariqoli:0.0,Shughni:0.0)HTU218:1.0,(ArabicAfroAsiatic:0.0,ModernHebrewAfroAsiatic:1.0)HTU314:0.0)HTU334:0.0)HTU354:0.0,((KataKati:1.0,Wakhi:0.0)HTU297:0.0,(AlbanianTosk:0.0,IronOssetic:1.0)HTU338:0.0)HTU357:1.0)HTU370:1.0)HTU372:1.0)HTU376:0.0)HTU381:0.0)HTU383:1.0)HTU385:1.0,(ScottishGaelic:1.0,(KachhiKoli/Kutchi:1.0,(Oadki:1.0,ThradriKoli:1.0)HTU296:0.0)HTU342:1.0)HTU387:0.0)HTU386:1.0)HTU384:1.0)HTU344:0.0)HTU329:0.0)HTU275:0.0)HTU245:0.0)HTU388:3.0)HTU195[488.0];
